.github/CODEOWNERS

```diff
@@ -95,18 +76,18 @@ xpti/ @tovinkere @andykaylor
 xptifw/ @tovinkere  @andykaylor
 
 # DPC++ tools
-llvm/**/file-table-tform/ @kbobrovs @AlexeySachkov
+llvm/**/file-table-tform/ @mlychkov @AlexeySachkov
 llvm/**/llvm-foreach/ @AlexeySachkov  @Fznamznon
 llvm/**/llvm-no-spir-kernel/ @AGindinson @AlexeySachkov
-llvm/**/sycl-post-link/ @kbobrovs @AlexeySachkov
-llvm/include/llvm/Support/PropertySetIO.h @kbobrovs @AlexeySachkov
-llvm/lib/Support/PropertySetIO.cpp @kbobrovs @AlexeySachkov
-llvm/unittests/Support/PropertySetIOTest.cpp @kbobrovs @AlexeySachkov
-llvm/lib/Support/Base64.cpp @kbobrovs @AlexeySachkov
+llvm/**/sycl-post-link/ @mlychkov @AlexeySachkov
+llvm/include/llvm/Support/PropertySetIO.h @mlychkov @AlexeySachkov
+llvm/lib/Support/PropertySetIO.cpp @mlychkov @AlexeySachkov
+llvm/unittests/Support/PropertySetIOTest.cpp @mlychkov @AlexeySachkov
+llvm/lib/Support/Base64.cpp @mlychkov @AlexeySachkov
 
 # Clang offload tools
-clang/tools/clang-offload-bundler/ @kbobrovs @sndmitriev
-clang/tools/clang-offload-wrapper/ @sndmitriev @kbobrovs
+clang/tools/clang-offload-bundler/ @mlychkov @sndmitriev
+clang/tools/clang-offload-wrapper/ @sndmitriev @mlychkov
 clang/tools/clang-offload-deps/ @sndmitriev
```

Comment:
```suggestion
clang/tools/clang-offload-deps/ @sndmitriev @mlychkov 
```
@mlychkov?

Reply:
I heard Mikhail is on vacation.
Tagging @AlexeySachkov .

Reply:
I'm not sure that either of us has enough expertise in clang offload tools to replace Konst by oneself, but probably if we both of us were added, then it should be fine.

Reply:
Added Mikhail.

Reply:
I think Alexey suggests adding @AlexeySachkov as well.

Reply:
Added Alexey as well.